rfcs/yyyymmdd-rfc-template.md

```diff
@@ -57,12 +57,15 @@ Make sure you’ve thought through and addressed the following sections. If a se
 * Platforms: does this work on all platforms supported by TensorFlow? If not, why is that ok? Will it work on embedded/mobile? Does it impact automatic code generation or mobile stripping tooling? Will it work with transformation tools?
 * Execution environments (Cloud services, accelerator hardware): what impact do you expect and how will you confirm?
 
-### Best Practices, Tutorials and Examples
+### Best Practices
 * Does this proposal change best practices for some aspect of using/developing TensorFlow? How will these changes be communicated/enforced?
+
+### Tutorials and Examples
 * If design changes existing API or creates new ones, the design owner should create end-to-end examples (ideally, a tutorial) which reflects how new feature will be used. Some things to consider related to the tutorial:
     - The minimum requirements for this are to consider how this would be used in a Keras-based workflow, as well as a non-Keras (low-level) workflow. If either isn’t applicable, explain why.
-    - It should show the usage of the new feature in an end to end example (from data reading to serving, if applicable). Many new features have unexpected effects in parts far away from the place of change that can be found by running through an end-to-end example.
-    - This should be written as if it is documentation of the new feature, i.e., consumable by a user, not a TensorFlow developer. The code does not need to work (since feature is not implemented yet).
+    - It should show the usage of the new feature in an end to end example (from data reading to serving, if applicable). Many new features have unexpected effects in parts far away from the place of change that can be found by running through an end-to-end example. TFX [Examples](https://github.com/tensorflow/tfx/tree/master/tfx/examples) have historically been good in identifying such unexpected side-effects and are as such one recommended path for testing things end-to-end.
+    - This should be written as if it is documentation of the new feature, i.e., consumable by a user, not a TensorFlow developer. 
+    - The code does not need to work (since the feature is not implemented yet) but the expectation is that the code does work before the feature can be launched or promoted. 
```

Comment:
What do you think about the use of "launched or promoted" here?  Should it be "merged" instead?

Reply:
Yes, good call.